frontend/src/components/common/content/ImageBlock.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { ContentImageProps } from "../../../types/ModuleEditorTypes";
+
+const TextBlock = ({
+  content,
+}: {
+  content: ContentImageProps;
+}): React.ReactElement => {
+  return <img src={content.link} alt="We should put an alt" />;
```

Comment:
Do we want captions in images? Don't think they were in any of the designs

Reply:
Good question. These content blocks will require more work, but I'll definitely ask design about this.